ui/uppy/src/components/PickerPanelTopBar.jsx

```diff
@@ -143,6 +142,20 @@ function PanelTopBar(props) {
             {props.i18n('addMore')}
           </button>
         )}
+        {!props.autoProceed && (
+          <button
+            className="uppy-dashboard-button-base uppy-dashboard-text-button"
+            type="button"
+            aria-label={props.i18n('uploadFiles')}
+            title={props.i18n('uploadFiles')}
+            disabled={props.isAllComplete && props.state.totalProgress === 100}
```

Comment:
This seems to disable the button until all are complete (isn't `isAllComplete && totalProgress == 100` redundant?); make sure the button is disabled whilst there are not uploads pending proceed.